helm/oauth2-proxy/values.yaml

```diff
@@ -259,4 +259,5 @@ metrics:
     # Prometheus scrape timeout
     scrapeTimeout: 30s
     # Add custom labels to the ServiceMonitor resource
-    labels: {}
+    labels:
+      prometheus: default
```

Comment:
This should probably be empty by default, don't you think?

Reply:
I asked for it as not to change the default behavior. That value came from here. 
https://github.com/oauth2-proxy/manifests/blob/main/helm/oauth2-proxy/values.yaml#L261
and here
https://github.com/oauth2-proxy/manifests/blob/main/helm/oauth2-proxy/templates/servicemonitor.yaml#L16